tests/RobotFramework/tests/configuration/lock_file.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Resource    ../../resources/common.resource
+Library    ThinEdgeIO
+
+Test Tags    
+Suite Setup    Setup
+Test Teardown   Get Logs
+
+*** Test Cases ***
+
+Check lock file existance in default folder
+    [Documentation]    Whe deploying thin-edge components in single-process containers, 
+    ...    the lock file mechanism used by some components (e.g. tedge-agent, tedge-mapper) 
+    ...    do not makes sense as the container filesystem is isolated. 
+    ...    Having the lock file system just adds unnecessary dependencies on the /run/lock folder.
+    File Should Exist    /var/lock/tedge-agent.lock
+    File Should Exist    /var/lock/tedge-mapper-c8y.lock
```

Comment:
The lock file path is `/run/lock/tedge-mapper-c8y.lock`.

Reply:
Furthermore, the path `/run` is configurable by `tedge config run.path set <path>`.